src/Makefile.am

```diff
@@ -9,12 +9,17 @@ endif
 
 if INCPOOL
 SUBDIRS += server pool
-else !INCPOOL
```

Comment:
I think it would be easier to just initialize `DIST_SUBDIRS` as
`DIST_SUBDIRS = utils replacements tds ctlib dblib odbc server pool apps`

Reply:
It's working! Can I change the patch accordingly?

Reply:
Sure! Thank you for you kind help :)